src/ERC20.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.6;
+
+import { IERC20 } from "./IERC20.sol";
+
+/// @title Modern and gas efficient ERC-20 implementation. 
+contract ERC20 is IERC20 {
+
+    string public override name;
+    string public override symbol;
+
+    uint8 public immutable override decimals;
+
+    uint256 public override totalSupply;
+
+    mapping(address => uint256) public override balanceOf;
+
+    mapping(address => mapping(address => uint256)) public override allowance;
+
+    constructor(string memory _name, string memory _symbol, uint8 _decimals) {
+        name     = _name;
+        symbol   = _symbol;
+        decimals = _decimals;
+    }
+
+    /*********************************/
+    /*** ERC-20 External Functions ***/
```

Comment:
Does the `ERC-20` prefix is needed here?

Reply:
Agreed. Remove.